### PR TITLE
Prettier still

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

updates prettier

## What is the value of this and can you measure success?

being pretty

### CODE REVIEW

File: tools/__tasks__/compile/hash/index.js [hunk 1]
```diff
@@ -106,7 +106,7 @@ module.exports = {
                             )
                         );
                     })
-                    .then(normalisedAssetMap => // save the asset map
+                    .then((normalisedAssetMap) => // save the asset map
```
Comment (SiAdcock) [inline tools/__tasks__/compile/hash/index.js]: Back to bracketing single params... but why in just this one instance?

Comment (sndrs) [inline tools/__tasks__/compile/hash/index.js]: I guess just still under dev? https://github.com/prettier/prettier/releases/tag/1.2.0

could raise that as a bug?

Comment (SiAdcock) [inline tools/__tasks__/compile/hash/index.js]: It's the comment that's causing prettier to want to parenthesise this parameter. I'll get a minimal repro working then raise an issue. I guess this shouldn't block this PR.

Comment (sndrs) [inline tools/__tasks__/compile/hash/index.js]: interesting...

Comment (SiAdcock) [inline tools/__tasks__/compile/hash/index.js]: Ah, to hell with it, it looks fine 😄

Comment (sndrs) [inline tools/__tasks__/compile/hash/index.js]: ¯\_(ツ)_/¯

Comment (sndrs) [inline tools/__tasks__/compile/hash/index.js]: that's +1 enough for me :)

**Review by SiAdcock — APPROVED** [review]

**Pull request merged**